src/liberror/specification

```diff
@@ -648,3 +648,27 @@ severity:normal
 ingroup:plugin
 module:storage
 see:9 75 109
+
+number:111
+description:Libgcrypt failed to initialize due to a version mismatch.
+severity:normal
+ingroup:plugin
+module:crypto
+
+number:112
+description:Failed to set up the crypto handle correctly.
+severity:normal
+ingroup:plugin
+module:crypto
+
+number:113
+description:Encryption failed due to an error reported by the backend.
+severity:normal
+ingroup:plugin
+module:crypto
+
+number:114
+description:Decryption failed due to an error reported by the backend.
+severity:normal
```

Comment:
I am unsure if failed decryption always should be an error. What if someone only wants access to a not encrypted key next to an encrypted one?

Maybe passing out the encrypted string makes sense then (because the application is not interested anyway).

Most likely both warning and error makes sense for *de*cryption.